_scripts/templates/Dart/makefile

```diff
@@ -10,6 +10,7 @@ SOURCES = $(GENERATED) \
 default: classes
 classes: $(SOURCES)
 	dart pub get
+	dart compile exe cli.dart
```

Comment:
These build steps generate executable file. You can use it in `run` target and add `classes` to `run`'s dependency. Same for Go makefile.

Reply:
I need to update the makefiles to work with real targets and dependencies and to this right. I created [this issue](https://github.com/antlr/grammars-v4/issues/2159) for general makefile-related issues. I didn't program the makefile very well because I was just following a "how to do Dart/Go" somewhere, and really just wanted to get something to work. The [ANTLR4 Runtime for Dart](https://github.com/antlr/antlr4/blob/master/doc/dart-target.md#antlr4-runtime-for-dart) didn't help much. For now, let's push the current changes in, and fix the makefiles in another step. -K